### ISSUE TITLE
Should not lint on box-imported and aliased R6 object instantiation.

### ISSUE THREAD

**Issue by radbasa** [Appsilon/box.linters#30]
## box.linters version

0.0.0.9006

## Source code to lint

```R
box::use(
  path/to/module_r6[class_alias = SomeClass],
)

s <- class_alias$new()
```

## Lint command

```R
lintr::lint(code, linters = lintr::linters_with_defaults(defaults = box.linters::rhino_default_linters))
```

## Lint result

```
<text>:6:6: warning: [box_usage_linter] <package/module>$function does not exist.
s <- class_alias$new()
     ^~~~~~~~~~~~~~~
```

## Expected result

Should not lint on `alias$new()`
